### DISCUSSION TITLE
🐛 [BUG] 饼图显示问题，当data中只有一个元素且不为首元素的值不为0时，渲染后的柄图中存在间隙

### DISCUSSION THREAD

**Discussion by sweetqianba** [antvis/G2Plot#3586] [Q&A] [answered]
### 🐛 饼图显示问题，当data中只有一个元素且不为首元素的值不为0时，渲染后的柄图中存在间隙

> Please provide a link by forking these links [G2Plot](https://codesandbox.io/s/g2plot-issue-template-gzfdq?file=/index.ts) or GitHub repo, a minimal reproduction.

* _Required_ **Link to minimal reproduction**:

### 📷 Step to reproduce
只有一个元素且不为首元素的值不为0时，渲染后的饼图中存在间隙

只有第一个元素不为0时，无此问题
### 🏞 Expected result
只有一个元素且不为首元素的值不为0时，渲染后的柄图中不存在间隙
### 🚑 Any additional [like screenshots]

* **G2Plot Version**: 2.4.25
* **Platform**:

**Comment by sweetqianba** [comment]
只有一个元素且不为首元素的值不为0时，渲染后的饼图中存在间隙
![image](https://user-images.githubusercontent.com/35681671/234752208-5ecf6fd4-fdc0-431a-a04f-f978d4505ba9.png)

**Comment by sweetqianba** [comment]
只有第一个元素不为0时，无此问题
![image](https://user-images.githubusercontent.com/35681671/234752366-7f2cbcbb-cb7c-4030-a777-f351b98e341f.png)

**Comment by hustcc** [comment, marked as answer]
符合预期的，原因是每个扇形都有一个默认的 lineWidth 和 stroke，就是白色的边框，即时数据为 0，扇形角度为 0，但是边框还是有的。

设置 lineWidth = 0 就不会有白线。

```ts
pieStyle: {
  lineWidth: 0,
}
```

为什么第一个为 1 的时候，就没有问题，因为第一个数据的扇形为 100%，他的边框直接盖住了其他扇形的边框。
